test/unit/rules/no-at-ember-render-modifiers-test.js

```diff
@@ -0,0 +1,73 @@
+import { MESSAGES } from '../../../lib/rules/no-at-ember-render-modifiers.js';
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'no-at-ember-render-modifiers',
+
+  config: true,
+
+  good: ['<div {{this.someModifier}}></div>', '<div {{someModifier}}></div>'],
```

Comment:
Looks like we need a valid test case for:
* `<div {{did-foo}}></div>`
* `{{did-insert}}` (by itself)

Reply:
done! thanks!